Specs/Scene/PointCloud3DTileContentSpec.js

```diff
@@ -436,6 +438,102 @@ defineSuite([
         });
     });
 
+    function countRenderedPixels(rgba) {
```

Comment:
Instead of having this here, maybe add a new expectation function, e.g.,
```javascript
expect(scene).toRenderPixelCount(5)
expect(scene).toRenderPixelCountGreaterThan(5)
```